js/index.js

```diff
@@ -206,31 +248,83 @@ function displayInit() {
     imagePromise = imagePromise.then(() => updateCustomStyles());
 }
 
-function clearBackgroundImage() {
+function defaultBackgroundImage() {
     const fileInput = document.getElementById('image-upload');
     fileInput.value = '';
-    delete appState.backgroundImage;
-    imageChanged = true;
+
+    // Remove solid background settings
+    delete appState.useSolidBackground;
+    delete appState.solidBackgroundColor;
+    delete appState.backgroundImage; // Correctly remove image
+
+    // Reset the inline styles on backgroundDiv
+    backgroundDiv.style.backgroundImage = '';  // Clear inline style
+    backgroundDiv.style.backgroundColor = '';  // Clear inline style
+
     save();
-    imagePromise = imagePromise.then(() => deleteImage(imageKey));
-    imagePromise = imagePromise.then(() => updateCustomStyles());
+	updateCustomStyles();  // Call this to apply the changes
+}
+
+function resetBackgroundColor() {
+    appState.gameAreaColor = "#1A1A1AFF"; // Reset to the default game area color
+    document.getElementById('color-input').value = appState.gameAreaColor;
+    save();
+    updateCustomStyles(); // Only updateCustomStyles is needed here, not init()
+}
+
+function removeImage() {
+    const fileInput = document.getElementById('image-upload');
+    fileInput.value = '';
+    delete appState.backgroundImage; // Remove the image URL
```

Comment:
Don't store a giant image url in localStorage! localStorage is limited to 10MB on most browsers, and it'll fail unexpectedly for a lot of people. There are also problems with not making it an Object url, since giant image files will get spit out in the HTML, creating weird lag while the browser churns through it figuring out what changed.

Reply:
Actually, I am likely confused due to the AI comment. This is a key into IndexedDB, not a url.

My point about the not using createObjectUrl still stands though (removed in a different section)